src/test/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/mapper/ActionPlanEntityMapperTest.kt

```diff
@@ -69,4 +72,30 @@ class ActionPlanEntityMapperTest {
     assertThat(actual).usingRecursiveComparison().isEqualTo(expected)
     verify(goalMapper).fromEntityToDomain(actionPlanEntity.goals!![0])
   }
+
+  @Test
+  fun `should map from entity summaries to domain summaries`() {
+    // Given
+    val summaryProjection1 = aValidActionPlanSummaryProjection(prisonNumber = aValidPrisonNumber())
+    val summaryProjection2 = aValidActionPlanSummaryProjection(prisonNumber = anotherValidPrisonNumber())
+    val expected = listOf(
+      aValidActionPlanSummary(
+        prisonNumber = summaryProjection1.prisonNumber,
+        reference = summaryProjection1.reference,
+        reviewDate = summaryProjection1.reviewDate,
+      ),
+      aValidActionPlanSummary(
+        prisonNumber = summaryProjection2.prisonNumber,
+        reference = summaryProjection2.reference,
+        reviewDate = summaryProjection2.reviewDate,
+      ),
+    )
+
+    // When
+    val actual = mapper.fromEntitySummariesToDomainSummaries(listOf(summaryProjection1, summaryProjection2))
+
+    // Then
+    assertThat(actual).hasSize(2)
```

Comment:
Functionally redundant (in addition to the assertion below), but the reason for failure is immediately clear.